src/components/errors/Error.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class Error extends React.Component {
+    render() {
+        const {
+            message,
+            statusCode,
+            createdAt,
+            severity,
+            recommendedActions,
+            pod,
+        } = this.props;
+
+        return (
+            <>
+                <div>Error</div>
+                <div>Error message: {message}</div>
+                <div>Status code: {statusCode}</div>
+                <div>Time of error: {createdAt}</div>
+                <div>Level of severity: {severity}</div>
+                <div>Recommended actions: {recommendedActions}</div>
+                <div>Pod: {pod}</div>
+            </>
+        )
+    }
+}
+
+export default Error;
```

Comment:
Add new line at EOF

Reply:
Addressed in my comment above—everyone should change their VS Code to automatically add in new lines at EOF.